boards/posix/native_posix/doc/board.rst

```diff
@@ -620,3 +628,26 @@ can feed it the following set of commands through a pipe:
    echo -e \
    'select kernel\nuptime\n!wait 500\nuptime\n!wait 1000\nuptime\n!quit' \
    | zephyr/zephyr.exe
+
+Host based flash access
+***********************
+
+In case a flash device is present the file system partitions on the flash device
+can be exposed through the host file system by enabling
+:option:`CONFIG_NATIVE_POSIX_FUSE_FLASH_ACCESS`.
+
+By default the partitions are exposed through the directory *flash* in the
+current working directory. This directory can be changed via the option
+command line option *--flash-mount*.
+
+Note that this feature requires a 32-bit version of the FUSE library on the host
```

Comment:
any version in particular you tested with and will maintain compatibility with?

Reply:
> Any version in particular you tested with and will maintain compatibility with?

The minimum version is 2.6 and this is enforced by the code (```define FUSE_USE_VERSION 26```)

> So if some part does require something in particular, it would be nice to document it clearly.

The only restriction is that a UNIX like OS (Linux, MacOS X, free BSD, ...) is used with a minimal FUSE version of 2.6. Note that Windows/WSL is not supported but FUSE is highly requested because of it widespread usage (e.g. GlusterFs).

Reply:
Ok. Would it make sense to mention this requirement in the doc?

Reply:
I could add it but dont know it is worth the effort as Zephyr only support WSL as build environment but WSL does not support 32-bit applications and as suche the native posix board is not supported on windows. This is in fact already clearly stated in the native posix board documentation.

Reply:
I meant the 2.6 version requirement, not the WSL thing.